core/src/impl/Kokkos_Profiling_C_Interface.h

```diff
@@ -254,17 +277,21 @@ struct Kokkos_Profiling_EventSet {
   Kokkos_Profiling_dualViewSyncFunction sync_dual_view;
   Kokkos_Profiling_dualViewModifyFunction modify_dual_view;
   Kokkos_Profiling_declareMetadataFunction declare_metadata;
-  char profiling_padding[11 * sizeof(function_pointer)];
+  Kokkos_Tools_transmitActionsFunction transmit_actions;
```

Comment:
provide_actions

Reply:
(maybe?)

Reply:
This is one I still prefer `transmit` for, tbh

Reply:
I don't like `transmit_actions` either

Reply:
As someone new to this, I really don't know what `transmit_actions` actually refers to (what actions are being transmitted, where are they being sent, etc.).

Reply:
Changed to "provide_tool_programming_interface" via 700baad